src/functions/template-tags/link.php

```diff
@@ -202,13 +202,13 @@ function tribe_get_listview_link( $term = null ) {
 }
 
 /**
- * Link to List View (Past)
+ * Returns a link to the general or category past view.
  *
- * Returns a link to the general or category past view
+ * @since 5.14.0
```

Comment:
This since here is wrong, this method exists before `5.14.0`.

Reply:
2.0.1

Reply:
@bordoni , this is fixed